packages/nodes-base/nodes/Aws/S3/GenericFunctions.ts

```diff
@@ -30,43 +30,24 @@ export async function awsApiRequest(
 	// tslint:disable-next-line:no-any
 ): Promise<any> {
 	const credentials = await this.getCredentials('aws');
-
-	const endpoint = new URL(
-		(((credentials.s3Endpoint as string) || '').replace('{region}', credentials.region as string) ||
-			`https://${service}.${credentials.region}.amazonaws.com`) + path,
-	);
-
-	// Sign AWS API request with the user credentials
-	const signOpts = {
-		headers: headers || {},
-		host: endpoint.host,
-		method,
-		path: `${endpoint.pathname}?${queryToString(query).replace(/\+/g, '%2B')}`,
-		body,
-	} as Request;
-	const securityHeaders = {
-		accessKeyId: `${credentials.accessKeyId}`.trim(),
-		secretAccessKey: `${credentials.secretAccessKey}`.trim(),
-		sessionToken: credentials.temporaryCredentials
-			? `${credentials.sessionToken}`.trim()
-			: undefined,
-	};
-
-	sign(signOpts, securityHeaders);
-
-	const options: OptionsWithUri = {
-		headers: signOpts.headers,
+	const requestOptions = {
+		qs: {
```

Comment:
There is something a bit strange happening with s3 node.

I tried to run `File` -> `Get All` to list all files from a bucket and this operation is now failing.